packages/docs/wrangler.jsonc

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
QwikUI is already using the new config, init?

Reply:
Yup, made the switch yesterday :)